example/main.go

```diff
@@ -7,8 +7,8 @@ func main() {
 }
 
 func rewrite() {
-	if handler.GetPath() == "/v1.0/hi" {
-		handler.SetPath("/v1.0/hello")
+	if handler.GetURI() == "/v1.0/hi" {
+		handler.SetURI("/v1.0/hello")
```

Comment:
maybe add `?name=panda` or something in these examples to make sure people know that this is not just the path